### PR TITLE
fix Close AppBar when tap

### LINKED ISSUE #120: Close AppBar when tap
In the mobile view,

- [ ]  AppBar should be closed when tapping on a button.
- [ ] AppBar should be closed when tapping on out of the AppBar.

### PR DESCRIPTION
**Pull request by furkay**
Fix #120 

I fixed 2 todo.

I wrap the menu focus, I got it to turn off on non-menu clicks and also when choosing from the menu.
close-appbar

### CODE REVIEW

**Comment by bgoktugozdemir** [comment]
Hi Furkan, thank you for your contribution. 

I had a little question. Can you answer that?

**Comment by furkay** [comment]
> Hi Furkan, thank you for your contribution.
> 
> I had a little question. Can you answer that?

Of course, what's the question?

**Comment by bgoktugozdemir** [comment]
> > Hi Furkan, thank you for your contribution.
> > I had a little question. Can you answer that?
> 
> Of course, what's the question?

I asked it above.

File: lib/navigation/ui/website_navigation.dart [hunk 1]
```diff
@@ -23,56 +23,83 @@ class WebsiteNavigation extends StatefulWidget {
 class _WebsiteNavigationState extends State<WebsiteNavigation> {
   bool _isContextMenuVisible = false;
 
+  final FocusNode menuFocusNode = FocusNode();
```
[the remark this answers is not in the document]
Comment (furkay) [reply lib/navigation/ui/website_navigation.dart:26]: We had to add focus to close the menu by capturing non-menu clicks. We needed to listen to the state of this focus. So I defined it here and called its listener in initstate.

[the remark this answers is not in the document]
Comment (bgoktugozdemir) [reply lib/navigation/ui/website_navigation.dart:26]: Oh, I see. So we added it to catch the clicks on the outside of the AppBar. Right?

[the remark this answers is not in the document]
Comment (furkay) [reply lib/navigation/ui/website_navigation.dart:26]: Yes, We need to put focus on the menu.

[the remark this answers is not in the document]
Comment (bgoktugozdemir) [reply lib/navigation/ui/website_navigation.dart:26]: Great 🎉  Thank you for the information.

**Review by bgoktugozdemir — APPROVED** [review]
Looks good to me. Great work. Thank you for your valuable contribution.

**Comment by bgoktugozdemir** [comment]
> Hello, I saw when the code went live, focusNode does not work as I expected in the mobile version. 
> I tried it as web in Responsive, it was not enough.
> You can reopen a task to have it closed on non-menu clicks.😕

Sure. I reopened this issue again.

**Pull request merged**